src/nuget/Microsoft.DotNet.BuildTools.nuspec

```diff
@@ -20,6 +20,7 @@
       <dependency id="Microsoft.DotNet.TestHost" version="1.0.3-prerelease" />
       <dependency id="Microsoft.DotNet.CoreCLR" version="1.0.0-prerelease" />
       <dependency id="Microsoft.DotNet.BuildTools.ApiTools" version="1.0.0-prerelease" />
+      <dependency id="NuProj" version="0.10.4-beta-gf7fc34e7d8" />
```

Comment:
If it is necessary it should be included in the BuildTools package itself. Please resolve your conflicts and you will see that I deleted all other dependencies in the nuspec.